src/app/pages/sessions/sessions/sessions.page.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-sessions',
+  templateUrl: './sessions.page.html',
+  styleUrls: ['./sessions.page.scss'],
+})
+export class SessionsPage implements OnInit {
+  type: string;
+  constructor(private activatedRoute: ActivatedRoute) {
+    this.activatedRoute.queryParamMap.subscribe(params => {
+      this.type = params.get('type');
+      if (this.type == null) {
```

Comment:
rather than using  if use ternary operator
this.type = this.type ? this.type : "all-sessions"